src/CheckBox/CheckBox.tsx

```diff
@@ -22,12 +23,12 @@ export const CheckBox: FC<CheckBoxProps> = ({
   errorMsg,
 }) => (
   <>
-    <BoxContainer id={id}>
+    <BoxContainer>
       <Text tag="span" typo="base" color={error ? 'red7' : 'secondary'}>
         {children}
       </Text>
 
-      <input type="checkbox" checked={checked} onChange={toggle} />
+      <input id={id} type="checkbox" checked={checked} onChange={toggle} />
```

Comment:
worth checking that this won't break any test? If some checkbox test where done looking for `id` and then finding the input inside moving it will beak them

Reply:
I've reverted back the change

Reply:
Yeah probably safest to keep the I'd where it is for now but we should move it to the input cause it's the right place